modules/concept-docs/pages/data-model.adoc

```diff
@@ -55,5 +55,23 @@ The [.api]`Map`, [.api]`List`, and [.api]`Queue` implementations may contain val
 * JsonObject
 * JsonArray
 
-The [.api]`Set` implementation may contain values of all of the above types except [.api]`JsonObject` and [.api]`JsonArray`.
+The `Set` implementation may contain values of all of the above types except `JsonObject` and `JsonArray`.
 
+
+== Using the CouchbaseMap Object
+
+If you need to feed a Couchbase document to an existing API that only works with `java.util.Map`, then CouchbaseMap may be a quick way to get started with tackling the task.
```

Comment:
Is that the use-case?  I thought it was for, I dunno, quick prototyping with Couchbase without having to learn subdoc properly.